### PR TITLE
Feature#3 task crud

### PR DESCRIPTION
**Pull request by jnegrete15**
#### We implemented CRUD actions to Taks model
 - Re-build of task model
 - set default vaules when a taks is created (user_id, status)
 - Basic html structure without styles yet

#### Create a task
Screenshot from 2021-05-05 18-12-38

#### Testing
1. run `rails s` command
2. Go to [localhost:3000/task/new](url)
3. there you go

### CODE REVIEW

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    respond_to do |format|
+      if @task.save
+        format.html { redirect_to @task, notice: "Task was successfully created." }
+        format.json { render :show, status: :created, location: @task }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @task.errors, status: :unprocessable_entity }
+      end
+    end
+  end
```
Comment (MiguelOcegueraM) [inline app/controllers/tasks_controller.rb:39]: nice

File: app/views/tasks/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+<p>
+  <strong>Due to:</strong>
+  <%= @task.due_to %>
+</p>
+
+<p>
+  <strong>User:</strong>
+  <%= @task.user_id %>
+</p>
```
Comment (MiguelOcegueraM) [inline app/views/tasks/show.html.erb:26]: love it

File: test/controllers/tasks_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+  test "should show task" do
+    get task_url(@task)
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get edit_task_url(@task)
+    assert_response :success
+  end
```
Comment (MiguelOcegueraM) [inline test/controllers/tasks_controller_test.rb:34]: nice one

File: test/system/tasks_test.rb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+  test "destroying a Task" do
+    visit tasks_url
+    page.accept_confirm do
+      click_on "Destroy", match: :first
+    end
+
+    assert_text "Task was successfully destroyed"
+  end
+end
```
Comment (MiguelOcegueraM) [inline test/system/tasks_test.rb:51]: is good to start with the test! congrats guys

**Review by MiguelOcegueraM — APPROVED** [review]
the code is very clean mates, like it

File: app/helpers/tasks_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module TasksHelper
+end
```
Comment (OchoaHub) [inline app/helpers/tasks_helper.rb:2]: Bro, if you don't need it, delete them :)

File: db/migrate/20210505212905_create_tasks.rb [hunk 6]
```diff
@@ -1,12 +1,10 @@
-# frozen_string_literal: true
-
 class CreateTasks < ActiveRecord::Migration[6.1]
   def change
     create_table :tasks do |t|
       t.string :name
       t.string :description
       t.boolean :status
-      t.datetime :due
+      t.datetime :due_to
```
Comment (OchoaHub) [inline db/migrate/20210505212905_create_tasks.rb:7]: I like this one

**Review by OchoaHub — APPROVED** [review]
Looks good for the task CRUD

**Pull request merged**